source/ios/AdaptiveCards/Podfile

```diff
@@ -1,5 +1,5 @@
 use_frameworks!
-pod 'MicrosoftFluentUI', '~> 0.1.9'
+pod 'MicrosoftFluentUI', '0.1.9'
```

Comment:
I'm not familiar with the ~> syntax, what does it mean?

Reply:
https://guides.cocoapods.org/syntax/podfile.html#tab_pod

<img width="583" alt="image" src="https://user-images.githubusercontent.com/16614499/190822038-15f01818-2c79-42a5-aa04-714d556b3565.png">